app/controllers/event1/UnauthPaymentSummaryController.scala

```diff
@@ -39,24 +40,42 @@ import views.html.event1.UnauthPaymentSummaryView
 import javax.inject.Inject
 
 class UnauthPaymentSummaryController @Inject()(
-                                        val controllerComponents: MessagesControllerComponents,
-                                        identify: IdentifierAction,
-                                        getData: DataRetrievalAction,
-                                        requireData: DataRequiredAction,
-                                        userAnswersCacheConnector: UserAnswersCacheConnector,
-                                        formProvider: UnauthPaymentSummaryFormProvider,
-                                        view: UnauthPaymentSummaryView
-                                 ) extends FrontendBaseController with I18nSupport {
+                                                val controllerComponents: MessagesControllerComponents,
+                                                identify: IdentifierAction,
+                                                getData: DataRetrievalAction,
+                                                requireData: DataRequiredAction,
+                                                userAnswersCacheConnector: UserAnswersCacheConnector,
+                                                formProvider: UnauthPaymentSummaryFormProvider,
+                                                view: UnauthPaymentSummaryView
+                                              ) extends FrontendBaseController with I18nSupport with Formatters {
 
   private val form = formProvider()
 
   def onPageLoad(waypoints: Waypoints): Action[AnyContent] = (identify andThen getData(EventType.Event1) andThen requireData) { implicit request =>
-    val mappedMemberOrEmployer = request.userAnswers
-      .getAll(MembersOrEmployersPage(EventType.Event1))(MembersOrEmployersSummary.readsMemberOrEmployer).zipWithIndex.map {
-      case (memberOrEmployerSummary, index) =>
+    val mappedMemberOrEmployer = getMappedMemberOrEmployer(request.userAnswers)
+    Ok(view(form, waypoints, mappedMemberOrEmployer, sumValue(request.userAnswers)))
+  }
 
-      val value = ValueViewModel(HtmlFormat.escape(memberOrEmployerSummary.unauthorisedPaymentValue.toString()).toString)
+  private def sumValue(userAnswers: UserAnswers) =
+    currencyFormatter.format(userAnswers.sumAll(MembersOrEmployersPage(EventType.Event1), MembersOrEmployersSummary.readsMemberOrEmployerValue))
+
+  def onSubmit(waypoints: Waypoints): Action[AnyContent] = (identify andThen getData(EventType.Event1) andThen requireData) {
+    implicit request =>
+      val mappedMemberOrEmployer = getMappedMemberOrEmployer(request.userAnswers)
```

Comment:
Move this down into formWithErrors big or change to a def/ lazy val

Reply:
moved under formWithErrors